### PR TITLE
[ENDPOINT] GET /organization/public OT246-23

### PR DESCRIPTION
**Pull request by ghnoob**
## Description

ref [OT246-23](https://alkemy-labs.atlassian.net/browse/OT246-23)

AS user
I WANT to see an Organization's data
SO THAT I have more knowledge about it.

### Approbation criteria: 

- A GET petition should be made to route /organization/public
- Should return name, image, phone and address

## Evidence

image

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -24,6 +25,7 @@ app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 
 app.use('/', indexRouter)
+app.use('/organization', organizationsRouter)
```
Comment (gabodmc) [inline app.js]: Esto va en el router index

**Review by gabodmc — CHANGES_REQUESTED** [review]
Esta muy bien, solo un cambio

**Review by gabodmc — APPROVED** [review]
Buenas correcciones. Aprobado!

**Review by kamp3r — APPROVED** [review]
Buenisimo rodri

**Review by tomasignacioc — APPROVED** [review]
joyaa

**Pull request merged**